Develop build fails. Platooning tactical plugin not able to find message type

### Types of Issue

<!--- What types of issue? Replace the space with an `x` in the box that applies: -->

- [x] Anomaly report (something appears to not work correctly)
- [ ] Enhancement request (describe the enhancement being requested)
- [ ] Other (please ensure the description clarifies why the issue doesn’t fall into either of the above categories)

### Descriptive summary
When trying to build carma-platform develop image the image cannot be built as the following error is thrown
```
In file included from /home/carma/carma_ws/src/carma-platform/platooning_tactical_plugin/src/platooning_tactical_plugin/platooning_tactical_plugin.cpp:1:0:
/home/carma/carma_ws/src/carma-platform/platooning_tactical_plugin/include/platooning_tactical_plugin/platooning_tactical_plugin.h:11:37: fatal error: cav_msgs/TrajectoryPlan.h: No such file or directory
compilation terminated.
carma-platform/platooning_tactical_plugin/CMakeFiles/platooning_tactical_plugin_library.dir/build.make:62: recipe for target 'carma-platform/platooning_tactical_plugin/CMakeFiles/platooning_tactical_plugin_library.dir/src/platooning_tactical_plugin/platooning_tactical_plugin.cpp.o' failed
CMakeFiles/Makefile2:24472: recipe for target 'carma-platform/platooning_tactical_plugin/CMakeFiles/platooning_tactical_plugin_library.dir/all' failed
make[2]: *** [carma-platform/platooning_tactical_plugin/CMakeFiles/platooning_tactical_plugin_library.dir/src/platooning_tactical_plugin/platooning_tactical_plugin.cpp.o] Error 1
make[1]: *** [carma-platform/platooning_tactical_plugin/CMakeFiles/platooning_tactical_plugin_library.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

```
It seems that the cav_msgs/TrajectoryPlan.h cannot be found by the platooning_tactical_plugin
### Expected behavior
Develop is always buildable
### Software version this applies to
<!--- what version are you running that you noticed this problem or deficiency? -->
Develop after 3.3.0 on commit 9a6b2f5
### Actual behavior
Develop fails to build with above error
### Steps to reproduce the behavior
From carma-platform/docker/
```
./build-image -d 
```

### Related work

Link to related tickets or prior related work here.

--- a/platooning_tactical_plugin/CMakeLists.txt
+++ b/platooning_tactical_plugin/CMakeLists.txt
@@ -153,10 +153,11 @@ add_executable(${PROJECT_NAME}_node src/platooning_tactical_plugin_node.cpp)
 
 ## Add cmake target dependencies of the executable
 ## same as for the library above
-add_dependencies(${PROJECT_NAME}_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_dependencies(${PROJECT_NAME}_node ${PROJECT_NAME}_library ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
 ## Specify libraries to link a library or executable target against
 target_link_libraries(${PROJECT_NAME}_node
+  ${PROJECT_NAME}_library
   ${catkin_LIBRARIES}
 )
 

--- a/platooning_tactical_plugin/CMakeLists.txt
+++ b/platooning_tactical_plugin/CMakeLists.txt
@@ -138,7 +138,7 @@ add_library(${PROJECT_NAME}_library src/${PROJECT_NAME}/platooning_tactical_plug
 ## Add cmake target dependencies of the library
 ## as an example, code may need to be generated before libraries
 ## either from message generation or dynamic reconfigure
-# add_dependencies(${PROJECT_NAME} ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_dependencies(${PROJECT_NAME}_library ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
 ## Declare a C++ executable
 ## With catkin_make all packages are built within a single CMake context

--- a/platooning_tactical_plugin/CMakeLists.txt
+++ b/platooning_tactical_plugin/CMakeLists.txt
@@ -115,7 +115,6 @@ catkin_package(
  INCLUDE_DIRS include
  LIBRARIES platooning_tactical_plugin
  CATKIN_DEPENDS cav_msgs roscpp rospy std_msgs autoware_msgs cav_srvs
- DEPENDS system_lib
 )
 
 ###########

--- a/platooning_tactical_plugin/src/platooning_tactical_plugin_node.cpp
+++ b/platooning_tactical_plugin/src/platooning_tactical_plugin_node.cpp
@@ -17,7 +17,7 @@
 #include <sstream>
 
 #include <ros/ros.h>
-#include "platooning_tactical_plugin/platooning_tactical_plugin.cpp"
+#include "platooning_tactical_plugin/platooning_tactical_plugin.h"
 
 int main(int argc, char** argv)
 {

